### PR TITLE
Add two more VirtualAgent examples.

### PR DESCRIPTION
**Pull request by sarahcstringer**
One for changing an agent's language, and one for turning on Sentiment Analysis.

### CODE REVIEW

File: twiml/voice/connect/virtualagent-3/virtualagent-3.6.x.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from twilio.twiml.voice_response import Connect, VoiceResponse, VirtualAgent
+
+response = VoiceResponse()
+connect = Connect()
+connect.virtualagent(connector_name='project', language='fr')
```
Comment (craigsdennis) [inline twiml/voice/connect/virtualagent-3/virtualagent-3.6.x.py:5]: This right?

File: twiml/voice/connect/virtualagent-3/virtualagent.6.x.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from twilio.twiml.voice_response import Connect, VoiceResponse, VirtualAgent
+
+response = VoiceResponse()
+connect = Connect(action='https://myactionurl.com/twiml')
+connect.virtualagent(
```
Comment (craigsdennis) [inline twiml/voice/connect/virtualagent-3/virtualagent.6.x.py:5]: Interesting...confused as to why there is no underscore in py

File: twiml/voice/connect/virtualagent-4/virtualagent-4.5.x.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+require 'twilio-ruby'
+
+response = Twilio::TwiML::VoiceResponse.new
+response.connect do |connect|
+    connect.virtual_agent(connector_name: 'project', sentiment_analysis: true)
```
Comment (craigsdennis) [inline twiml/voice/connect/virtualagent-4/virtualagent-4.5.x.rb:5]: Yet there is in ruby

**Review by craigsdennis — APPROVED** [review]
LGTM if it works!

**Comment by sarahcstringer** [comment]
Fixed the Python and I went through and tested all the other code samples to make sure they're working.

**Pull request merged**